nemo_skills/pipeline/convert.py

```diff
@@ -165,9 +172,9 @@ def convert(
         pass
 
     # TODO: add support for qwen nemo conversion
-    if model_type == "qwen":
-        if convert_from == "nemo" or convert_to == "nemo":
-            raise ValueError("NeMo conversion for Qwen models is not supported yet")
+    # if model_type == "qwen":
```

Comment:
Let's keep the if, but only for the from part, since that's still not supported

Reply:
ok, got it. I can add a comment then